data_tools/compute_coding_progress.py

```diff
@@ -6,36 +6,37 @@
 
 if (len(sys.argv) != 2):
     print ("Usage python compute_coding_progress.py coda_crypto_token")
-    exit(1)
+        exit(1)
 
 CODA_CRYPTO_TOKEN_PATH = sys.argv[1]
-
 fcw.init_client(CODA_CRYPTO_TOKEN_PATH)
-data = {}
-
-ids = fcw.get_dataset_ids()
-
-data['coding_progress'] = {}
 
-for id in ids:
-    data['coding_progress'][id] = {}
-    messages = []
-    messages_with_labels = 0
-
-    # New scheme
-    metrics = fcw.get_dataset_metrics(id)
-    if metrics != None:
-        data['coding_progress'][id] = metrics   
-        continue
-        
+def compute_coding_progress(ids,messages,metrics):
```

Comment:
spaces between variable names in the arg list please:

ids, messages, metrics

And please can you add a doc comment saying what they are? See some of the code in CoreDataModules for an example